louiseA/file-1.js

```diff
@@ -0,0 +1,103 @@
+1.
```

Comment:
Take some time and look over the below comments, and then at the homework solutions for more clarification. Don't forget to take some time to explain your reasoning in your answers, as that is a vital part to solidifying your own understanding of the material.